packages/hop-node/src/watchers/HealthCheckWatcher.ts

```diff
@@ -129,6 +133,22 @@ type InvalidBondWithdrawal = {
   timestamp: number
 }
 
+type UnrelayedTransfers = {
+  transactionHash: string
+  token: string
+  recipient: string
+  destinationChainId: number
+  amount: BigNumber
+  relayer: string
+  relayerFee: BigNumber
+}
+
+type UnsetTransferRoots = {
```

Comment:
This entity type should be named as singular and not plural: `UnsetTransferRoots` => `UnsetTransferRoot`